packages/fxa-settings/src/components/Settings/FlowRecoveryKeyConfirmPwd/index.tsx

```diff
@@ -14,13 +14,15 @@ import {
   AuthUiErrorNos,
   AuthUiErrors,
   composeAuthUiErrorTranslationId,
+  getLocalizedErrorMessage,
 } from '../../../lib/auth-errors/auth-errors';
 import InputPassword from '../../InputPassword';
 import { LockImage } from '../../images';
 import Banner, { BannerType } from '../../Banner';
 import { RecoveryKeyAction } from '../PageRecoveryKeyCreate';
 import { Link } from '@reach/router';
 import { HomePath } from '../../../constants';
+import sentryMetrics from 'fxa-shared/lib/sentry';
```

Comment:
Is sentry used in here?

Reply:
Not anymore! Thanks for spotting. Also removed a few other unused imports.